integration_test/gce/gce_testing.go

```diff
@@ -413,6 +436,16 @@ func nonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) (*mon
 	}
 }
 
+// firstTrace evaluates the given iterator, returning its first trace,
+// or nil if the iterator is empty.
+func firstTrace(it *trace.TraceIterator) *cloudtrace.Trace {
+	trace, err := it.Next()
+	if err == iterator.Done {
```

Comment:
The only [documented](https://pkg.go.dev/cloud.google.com/go/trace@v1.3.0/apiv1#TraceIterator.Next) error here is `iterator.Done`, but I can add that check in just in case.